src/main/java/com/hubspot/smtp/messages/InputStreamMessageContent.java

```diff
@@ -0,0 +1,27 @@
+package com.hubspot.smtp.messages;
+
+import java.io.InputStream;
+
+import io.netty.handler.stream.ChunkedStream;
+
+public class InputStreamMessageContent extends MessageContent {
+  private final ChunkedStream chunkedStream;
+  private final int size;
+
+  public InputStreamMessageContent(InputStream stream, int size, boolean applyDotStuffing) {
```

Comment:
rather than passing an InputStream, I think we should use guava's InputStreamSupplier or `Supplier<InputStream>`. It's odd that we'll take something that we close without any way of opening it.